development/tools/cmd/externalsecretschecker/README.md

```diff
@@ -0,0 +1,23 @@
+# External secrets checker
+
+## Overview
+
+This command checks external secrets synchronization status, and if every secret have corresponding external secret.
+
+## Usage
+
+To run it, use:
+
+```bash
+go run main.go --kubeconfig=/Users/i542853/.kube/config --ignored-secrets "namespace/secretName,namespace/secretName2"
+```
+
+### Flags
+
+See the list of available flags:
+
+| Name                      | Required | Description                                                                                             |
+| :------------------------ | :------: | :------------------------------------------------------------------------------------------------------ |
+| **--kubeconfig**          |   Yes    | The path to the `kubeconfig` file, needed to connect to a cluster.                                      |
+| **--ignored-secrets**     |    No    | List of ignored secrets. List contains of secrets in `namespace/secretName` format, separated by comma. |
+| **--namespaces**          |    No    | List of analyzed namespaces. The program scans all namespaces if the list is empty.                     |
```

Comment:
```suggestion
| **--namespaces**          |    No    | The list of analyzed namespaces. The program scans all namespaces if the ignored secrets list is empty.                     |
```
Did you mean the ignored secrets list?
Just to make sure: Does the program scan all namespaces if the ignored secrets list is (totally) empty or all namespaces for which there aren't any ignored secrets on the list?

Reply:
The program scans all namespaces unless a list of namespaces to scan is provided through `namespaces` flag.